.scalafmt.conf

```diff
@@ -1,8 +1,8 @@
 align.openParenCallSite = false
 align.openParenDefnSite = false
 continuationIndent.defnSite = 2
-danglingParentheses = true
-docstrings = JavaDoc
+danglingParentheses.preset = true
+docstrings.style = Asterisk
```

Comment:
not really sure if  release note or 'incompatible change' action is necessary
in newer versions of scalafmt properties are a bit different